libmscore/figuredbass.cpp

```diff
@@ -1770,6 +1773,256 @@ FiguredBass* Score::addFiguredBass()
+                        harmony = native_pitch + int(AccidentalVal::FLAT);
+                        break;
+                  case FiguredBassItem::Modifier::NATURAL:
+                        harmony = native_pitch;
+                        break;
+                  case FiguredBassItem::Modifier::SHARP:
+                        harmony = native_pitch + int(AccidentalVal::SHARP);
+                        break;
+                  case FiguredBassItem::Modifier::DOUBLESHARP:
+                        harmony = native_pitch + int(AccidentalVal::SHARP);
+                        harmony = native_pitch + int(AccidentalVal(modifier));
+                        break;
+                  case FiguredBassItem::Modifier::CROSS:
+                  case FiguredBassItem::Modifier::BACKSLASH:
+                        harmony = native_pitch + int(acciv) + 1; // adding 1 semi tone, flat->natural, natural->sharp, etc.
+                        break;
+                  case FiguredBassItem::Modifier::SLASH:
+                  case FiguredBassItem::Modifier::NUMOF:
+                  default:
+                        qDebug("measure #%d Modifier = %hhd not yet implemented", segment->measure()->no(), modifier);
+                        return;
+            }
+            harmony += octave;
+            for ( int h : harmonys )
+                  while ( h >= harmony )
+                        harmony += 12;
+            harmonys.push_back(harmony);
+      };
+      for (Element* e : segment->annotations()) {
+            if (e->type() == Element::Type::FIGURED_BASS ) {
+                  FiguredBass *fb = static_cast<FiguredBass*>(e);
+                  std::vector<int> harmony;
+                  std::vector<FiguredBassItem*> items = fb->getFiguredBaseItems();
+                  sort( items.begin(), items.end(), [&](FiguredBassItem *i1, FiguredBassItem *i2) {
+                        return get_digit(i1->digit()) < get_digit(i2->digit());
+                  });
+                  FiguredBassItem::Modifier modifier = (none == items[0]->prefix()) ? items[0]->suffix(): items[0]->prefix();
+                  switch ( items.size() ) {
+                        case 0:
+                              render_harmony(none, 3, none);
+                              render_harmony(none, 5, none);
+                              break;
+                        case 1: {
+                              // a single figured bass missing a number, just with a symbol, implies the digit 3.
+                              int digit = (FBIDigitNone==items[0]->digit()) ? 3 : items[0]->digit();
+                              switch (digit) {
+                                    case 2:
+                                          // render_relative({2,4,6});
+                                          render_harmony(modifier,2,none);
+                                          render_harmony(none, 4, none);
+                                          render_harmony(none, 6, none);
```

Comment:
Well, it might be possible that `2` alone had been used by some author to indicate `2 4 6` (see comment above about 'always'), but for that meaning at least `2 4`, if not the full `2 4 6`, has to be expected as most common. No promises, but I'll try to look at the scores I have at hand for occurrences of this.

And, thanks but no, I am not a Wikipedia contributor and I would prefer not to open another front...